compiler/rustc_middle/src/ty/sty.rs

```diff
@@ -1436,7 +1436,7 @@ pub struct ConstVid<'tcx> {
 rustc_index::newtype_index! {
     /// A **region** (lifetime) **v**ariable **ID**.
     #[derive(HashStable)]
-    #[debug_format = "'_#{}r"]
+    #[debug_format = "'?{}"]
```

Comment:
The `'_#` prefix had the advantage of being closer to correct syntax than `'?`: text editors used to be able to do syntax coloring on a .mir file as if it were a rust file :)

Reply:
😅 Maybe we should format MIR dumps separately, and not rely on the display impl of RegionVid for that...

Reply:
Also maybe we should just make tools for syntax highlighting MIR dumps 🤔